numaprom/udf/postprocess.py

```diff
@@ -147,25 +148,103 @@ def _publish(final_score: float, payload: StreamPayload) -> List[bytes]:
     return [publisher_json]
 
 
+def _calculate_ensemble_score(payload: StreamPayload, metric_config: MetricConf) -> float:
+    # Load config
+    metric_config = get_metric_config(
+        metric=payload.composite_keys["name"], namespace=payload.composite_keys["namespace"]
+    )
+
+    # Get static scores
+    static_clf = StaticThreshold(upper_limit=metric_config.static_threshold)
+    static_scores = static_clf.score_samples(payload.get_stream_array())
+    mean_static_score = np.mean(static_scores)
+
+    # Get model scores
+    model_scores = payload.get_stream_array()
+    mean_model_score = np.mean(model_scores)
+
+    # Perform score normalization
+    postproc_clf = TanhNorm()
+    norm_static_score = postproc_clf.transform(mean_static_score)
+    norm_model_score = postproc_clf.transform(mean_model_score)
+
+    # Calculate ensemble score
+    static_wt = metric_config.static_threshold_wt
+    model_wt = 1.0 - metric_config.static_threshold_wt
+    ensemble_score = (static_wt * norm_static_score) + (model_wt * norm_model_score)
```

Comment:
Also, if you are setting default static weight to 0, we would not need separate methods to calculate scores right?